docs/pages/database-access/guides/azure-postgres-mysql.mdx

```diff
@@ -3,7 +3,7 @@ title: Database Access with Azure PostgreSQL and MySQL
 description: How to configure Teleport database access with Azure Database for PostgreSQL and MySQL.
 ---
 
-(!docs/pages/includes/database-access/db-introduction.mdx  dbType="Azure PostgreSQL or MySQL" dbConfigure="Azure PostgreSQL or MySQL database with IAM authentication" dbName="Azure PostgreSQL or MySQL" !)
+(!docs/pages/includes/database-access/db-introduction.mdx dbType="Azure PostgreSQL or MySQL" dbConfigure="with IAM authentication"!)
```

Comment:
`dbConfigure` should be `with Microsoft Entra ID-based authentication` or `with Microsoft Entra ID for authentication`
https://learn.microsoft.com/en-us/azure/postgresql/single-server/how-to-configure-sign-in-azure-ad-authentication

Same applies to Azure Cache for Redis:
https://learn.microsoft.com/en-us/azure/azure-cache-for-redis/cache-azure-active-directory-for-authentication

I believe we already updated the "how it works" section below on this though.